src/views/Login/Login.vue

```diff
@@ -189,12 +197,18 @@ export default {
             this.$router.push('/change-password');
           } else {
             Promise.all([
-              this.$store.dispatch('userManagement/getUsers'),
```

Comment:
LDAP user without group name and privilege is able to login. Not expected, It is because we have removed the getUsers. Need to find a solution.